src/test/java/uk/gov/hmcts/sptribs/caseworker/event/page/IssueDecisionSelectRecipientsTest.java

```diff
@@ -39,14 +39,14 @@ void shouldBeSuccessfulForValidRecipients() {
         final AboutToStartOrSubmitResponse<CaseData, State> response = selectRecipients.midEvent(caseDetails, caseDetails);
 
         //Then
-        assertThat(response.getData().getCaseIssueDecision().getRecipientSubject().equals(SubjectCIC.SUBJECT));
+        assertThat(response.getData().getCaseIssueDecision().getRecipients().equals(ContactPartiesCIC.SUBJECTTOCONTACT));
     }
```

Comment:
As above, use == to test equality with an enum.

Reply:
getRecipients() returns set so not possible to use ==. Using == will result in syntax error

Reply:
OK. If it returns a Set<ContactPartiesCIC>, instead of equals, use contains?